src/internal.c

```diff
@@ -5004,7 +5004,13 @@ int EccVerify(WOLFSSL* ssl, const byte* in, word32 inSz, const byte* out,
 #endif /* WOLFSSL_ASYNC_CRYPT */
     {
         if (ret != 0 || ssl->eccVerifyRes == 0) {
+#ifdef WOLFSSL_STSAFEA100
```

Comment:
I am okay with the way it was, but if we keep this I would prefer a new generic macro like ‘WOLFSSL_PK_PASS_ERROR’. This is not STSafe specific. There is a place in RSA verify like this too that should get changed.

Reply:
Based on feedback, will revert to original change here (i.e. remove conditional-compilation based on WOLFSSL_STSAFEA100).